File: Ejercicios Propuestos 1/Ejercicio 2/Ejercicio 2.java
import java.util.Scanner;
public class Ejercicio2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese un número límite: ");
        int n = sc.nextInt();
        int sumaPares = 0, sumaImpares = 0, sumaMult5 = 0;
        for (int i = 1; i <= n; i++) {
            if (i % 2 == 0) sumaPares += i;
            else sumaImpares += i;   
            if (i % 5 == 0) sumaMult5 += i;
        }
        System.out.println("Suma de pares: " + sumaPares);
        System.out.println("Suma de impares: " + sumaImpares);
        System.out.println("Suma de múltiplos de 5: " + sumaMult5);
    }
}

File: Ejercicios Propuestos 1/Ejercicio 2/Ejercicio 2.jl
suma_pares = 0
suma_impares = 0
suma_multp5 = 0

print("Ingrese un número entero (mayor que 1): ")
n = parse(Int, readline())

while n <= 1
    println("ERROR: El número debe ser estrictamente mayor que 1.")
    print("Por favor, intente de nuevo: ")
    global n = parse(Int, readline())
end

for i in 1:n
    if i % 2 == 0
        global suma_pares += i
    else
        global suma_impares += i
    end
    
    if i % 5 == 0
        global suma_multp5 += i
    end
end

println("La suma de números pares entre 1 y ", n, " es: ", suma_pares)
println("La suma de números impares entre 1 y ", n, " es: ", suma_impares)
println("La suma de números múltiplos de 5 entre 1 y ", n, " es: ", suma_multp5)
